assets/app/data/crowbar/services/crowbar.factory.js

```diff
@@ -0,0 +1,84 @@
+(function() {
+
+    angular
+        .module('suseData.crowbar')
+        .factory('crowbarFactory', crowbarFactory);
+
+    crowbarFactory.$inject = ['$q', '$http', 'COMMON_API_V2_HEADERS'];
+    /* @ngInject */
+    function crowbarFactory($q, $http, COMMON_API_V2_HEADERS) {
+        var factory = {
+            getEntity: getEntity,
+            getRepositoriesChecks: getRepositoriesChecks,
+            upgrade: upgrade,
+            getUpgradeStatus: getUpgradeStatus
+        };
+
+        return factory;
+
+        /**
+         * Get Crowbar Entity
+         * 
+         * @return {Promise}
+         */
+        function getEntity() {
+
+            var requestOptions = {
+                method: 'GET',
+                url: '/api/crowbar',
+                headers: COMMON_API_V2_HEADERS
```

Comment:
That should be isolated to crowbar endpoints only, so we allow different headers for different endpoints in the future (OpenStack, for instance)
Maybe as part of `suseData.crowbar` specific configuration... What do you think?